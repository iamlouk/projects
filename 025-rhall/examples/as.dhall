{ foo = (42 as Any) as Int, bar = (true as Any) as Text }
