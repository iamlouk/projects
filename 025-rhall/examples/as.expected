{ foo = Some(42), bar = None(Text) }
